src/test/scala/com/itechart/ny_accidents/service/metric/WeatherMetricServiceTest.scala

```diff
@@ -1,74 +1,75 @@
 package com.itechart.ny_accidents.service.metric
 
 import com.google.inject.Guice
-import com.itechart.ny_accidents.{GuiceModule, TestSparkApi}
 import com.itechart.ny_accidents.entity.{District, MergedData, ReportMergedData, WeatherForAccident}
-import org.apache.spark.broadcast.Broadcast
+import com.itechart.ny_accidents.{GuiceModule, TestSparkApi}
 import org.apache.spark.rdd.RDD
 import org.scalatest.FunSpec
 
-import scala.collection.immutable
 
-/*
 class WeatherMetricServiceTest extends FunSpec {
 
 
   private val injector = Guice.createInjector(new GuiceModule)
   private val weatherMetricService = injector.getInstance(classOf[WeatherMetricService])
 
   describe("Reports") {
-    it("should return correct map of values grouping by phenomenon") {
-      val data: RDD[ReportMergedData] = TestSparkApi.spark.parallelize(Seq(
-        ReportMergedData(null, None, Some(WeatherForAccident(.0, .0, .0, "A", .0, .0))),
-        ReportMergedData(null, None, Some(WeatherForAccident(.0, .0, .0, "B", .0, .0))),
-        ReportMergedData(null, None, Some(WeatherForAccident(.0, .0, .0, "C", .0, .0))),
-        ReportMergedData(null, None, Some(WeatherForAccident(.0, .0, .0, "D", .0, .0)))
+    it("should return correct values grouping by phenomenon") {
+      val data: RDD[MergedData] = TestSparkApi.spark.parallelize(Seq(
+        MergedData(null, None, Some(WeatherForAccident(.0, .0, .0, "A", .0, .0))),
+        MergedData(null, None, Some(WeatherForAccident(.0, .0, .0, "B", .0, .0))),
+        MergedData(null, None, Some(WeatherForAccident(.0, .0, .0, "C", .0, .0))),
+        MergedData(null, None, Some(WeatherForAccident(.0, .0, .0, "D", .0, .0)))
       ))
 
-      val expectedResult: Map[String, Double] = Map[String, Double](
-        "A" -> 25.0,
-        "B" -> 25.0,
-        "C" ->  25.0,
-        "D" -> 25.0)
-      val result: Map[String, Double] = weatherMetricService.getPhenomenonPercentage(data).collect().toMap
+      val expectedResult: Seq[(String, Int, Double)] = Seq (
+        ("A", 1, 25.0),
+        ("B", 1, 25.0),
+        ("C", 1, 25.0),
+        ("D", 1, 25.0)).sortBy(_._1)
```

Comment:
`.sortBy(_._1)` - please don't put any extra computations/logic into test. Just hard code expected value as is. Tests should straightforward as much as possible.

Reply:
But what if the order of the objects in the collection is not important to me?

Reply:
Ok, then why do you sort this collection? The idea is try keep tests simple. For instance, you expect that some func return Array that should be sorted. Instead of this 
```
val expected = [3,2,1,78,4,0].sort()
```
it's much more better use already evaluated value:
```
val expected = [0,1,2,3,4,78]
```
then reviewer don't have to take some extra effort to compute this value in mind.

Reply:
Ok, fixed